### ISSUE TITLE
Loading icon is strange

### ISSUE THREAD

**Issue by cassidyjames** [system76/pop-icon-theme#15]
I've thought this and it's been mentioned by others: the loading icon is kind of a strange and uncommon metaphor. We should probably look at using a more common/standard loading spinner.

**Comment by jbicha** [comment]
You mentioned that it's a "spinner". Well it wasn't spinning when I saw it in Pop!_Shop.

**Comment by cassidyjames** [comment]
@jbicha that is indeed strange. You were talking about this icon, though?

![screenshot from 2017-09-29 15-06-35](https://user-images.githubusercontent.com/611168/31036105-16f458d6-a528-11e7-9f2e-5d30cbc25e54.png)


Here it's definitely spinning when loading.

**Comment by jbicha** [comment]
Yes, it looked stationary to me.

**Comment by WatchMkr** [comment]
@jbicha we think that's because VMs disable animations so you weren't seeing it spin. 

After more discussion we agreed it's placement is out of context and we're considering removing the spinner in the header bar and adding it where items are loading for better contextual feedback. We'll talk with the elementary folks to see if they agree as well.

**Comment by jbicha** [comment]
Yes, I was using VirtualBox since my primary OS is regular Ubuntu 17.10.

**Comment by cassidyjames** [comment]
@jbich @WatchMkr I can confirm that GTK animations are disabled across the board in VirtualBox.
